drop-transfer/src/manager.rs

```diff
@@ -998,6 +1004,46 @@ pub(crate) async fn restore_transfers_state(state: &Arc<State>, logger: &Logger)
     *state.transfer_manager.outgoing.lock().await = outgoing;
 }
 
+pub(crate) async fn resume(
+    refresh_trigger: tokio::sync::watch::Receiver<()>,
```

Comment:
It's a minor but you don't need the full ownership here of the `Receiver`, meaning it can be passed by reference, as the stop token is for example